<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsSourceFieldsProperties</class>
 <widget class="QWidget" name="QgsSourceFieldsProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTableWidget" name="mFieldsList"/>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item row="0" column="4">
      <widget class="QToolButton" name="mCalculateFieldButton">
       <property name="toolTip">
        <string>Field calculator</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="mToggleEditingButton">
       <property name="toolTip">
        <string>Toggle editing mode</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="mDeleteAttributeButton">
       <property name="toolTip">
        <string>Delete field</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="shortcut">
        <string>Ctrl+X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="mAddAttributeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>New field</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="shortcut">
        <string>Ctrl+N</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="mSaveLayerEditsButton">
       <property name="toolTip">
        <string>Save Layer Edits</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mFieldsList</tabstop>
  <tabstop>mAddAttributeButton</tabstop>
  <tabstop>mDeleteAttributeButton</tabstop>
  <tabstop>mToggleEditingButton</tabstop>
  <tabstop>mSaveLayerEditsButton</tabstop>
  <tabstop>mCalculateFieldButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
